src/components/Sidebar/UserInfo/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect, useContext } from 'react';
+import PulseLoader from 'react-spinners/PulseLoader';
+import HomeButton from './HomeButton';
+import LogOutButton from './LogOutButton';
+
+import constants from '../../../constants';
+import { LoginContext } from '../../../contexts';
+import { getUserInfo } from '../../../utils';
+import { IUserInfo } from '../../../interfaces';
+
+export default function UserInfo() {
+  const context = useContext(LoginContext);
+
+  const [userInfo, setUserInfo] = useState<IUserInfo>({
+    username: 'N/A',
+    registeredOn: 'N/A',
+    totalMessages: 0,
+    balance: 0
+  });
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+
+  useEffect(() => {
+    function update(address: string): void {
+      getUserInfo(address)
+        .then(data => {
+          setUserInfo(data);
+          setIsLoading(false);
+        })
+        .catch(() => {
+          setIsLoading(false);
+        });
+    }
+
+    if (context.user) {
+      update(context.user.address);
+
+      const interval = setInterval(() => {
+        update(context.user.address);
+      }, 10000);
+
+      return () => {
+        clearInterval(interval);
+      };
+    }
+  }, [context]);
+
+  return (
+    <div className="pl-3 pr-3 pb-3 pt-2">
+      <LoginContext.Consumer>
+        {context => (
+          <>
```

Comment:
Remove this empty tag.

Reply:
This is short syntax for a React Fragment and is necessary to return multiple elements without a parent.